dsa-solutions/lc-solutions/0000-0099/0019-remove-nth-node-from-end-of-list.md

```diff
@@ -0,0 +1,177 @@
+---
+id: remove-nth-node-from-end-of-list
+title: Remove nth node from end of list (LeetCode)
+sidebar_label: 0019-remove-nth-node-from-end-of-list
+tags:
+  - Two Pointers
+  - Linked List
+description: Given the head of a linked list, remove the nth node from the end of the list and return its head.
```

Comment:
replace 

`description: Given the head of a linked list, remove the nth node from the end of the list and return its head.`

to

`description: "Given the head of a linked list, remove the nth node from the end of the list and return its head."`